stylesheets/components/image.css

```diff
@@ -0,0 +1,35 @@
+@import url("/stylesheets/main.css");
+
+.image-container{
+    margin: 1rem;
+}
+
+.image{
+    max-height: 400px;
+    max-inline-size: 100%;
+    block-size: auto;
+    object-fit: cover;
+}
+
+.image-square-round{
+    border-radius: 5%;
+}
+
+.image-round{
+ 
```

Comment:
Remove blank space.Other than the code is clean and looks pretty good.

Reply:
Done as suggested , thankyou.